pylint/lint.py

```diff
@@ -726,6 +734,18 @@ def check(self, files_or_modules):
         """main checking entry: check a list of files or modules from their
         name.
         """
+        temp_files_or_modules = []
```

Comment:
Thanks Caludiu. I will look into Radoslaw's Solution and work on it.

Regards,
Rohit

On Tue, Feb 23, 2016 at 8:24 PM, Claudiu Popa <notifications@github.com>
wrote:

> In pylint/lint.py
> <https://github.com/PyCQA/pylint/pull/824#discussion_r53790108>:
>
> > @@ -726,6 +734,18 @@ def check(self, files_or_modules):
> >          """main checking entry: check a list of files or modules from their
> >          name.
> >          """
> > +        temp_files_or_modules = []
>
> This should be already handled by this PR
> https://bitbucket.org/logilab/astroid/pull-requests/80/add-option-to-search-for-all-python-files/diff.
> Basically you'll have to pass list_all to get_module_files in order to get
> all the files. You can see what Radoslaw did and try to figure it out from
> there (
> https://bitbucket.org/marvdreamwalker/pylint/commits/1221a75545c06fda64c2453c63f218f33cca8104,
> he never did the PR unfortunately and disappeared after the sprint we had).
> The idea is that this should be controlled by a recursive flag, since I
> don't want to have a change like this by default.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PyCQA/pylint/pull/824/files#r53790108>.
>